creator-node/src/routes/files.js

```diff
@@ -71,10 +69,12 @@ const streamFromFileSystem = async (req, res, path) => {
 
       // Add a content range header to the response
       res.set('Content-Range', formatContentRange(start, end, stat.size))
+      res.set('Content-Length', end - start)
```

Comment:
is this off by 1? 
im wondering if it should be (end - start) + 1

in the ex on https://developer.mozilla.org/en-US/docs/Web/HTTP/Range_requests

```
HTTP/1.1 206 Partial Content
Content-Range: bytes 0-1023/146515
Content-Length: 1024
...
(binary content)
```

here start would be 0, end would be 1023, so the CL we would return is 1023 instead of 1024 in the example

Reply:
yeah, the byte range is inclusive [0,1023] if im reading right

Reply:
Yep you're right, nice catch